blocks/research/research.css

```diff
@@ -0,0 +1,21 @@
+.research {
+  display: flex;
+  background: rgb(242, 242, 242);
+  padding: 78px 160px 70px 130px;
+  justify-content: center;
+  gap: 73px;
+}
+@media screen and (max-width: 660px) {
+  .research {
+    display: flex;
+    background: rgba(255, 255, 255, 1);
+    padding: 54px 25px 0 16px;
+    justify-content: space-between;
```

Comment:
@Woolfs1423 You can remove it as you already have a flex gap set to define space between the flex items 😉